ld.lld: error: undefined hidden symbol: __guard_local
seen on at least i386, sparc64

Index: tests/modules/Makefile
--- tests/modules/Makefile.orig
+++ tests/modules/Makefile
@@ -101,7 +101,7 @@ all: $(TEST_MODULES)
 	$(CC) -I../../src $(CFLAGS) $(SHOBJ_CFLAGS) -fPIC -c $< -o $@
 
 %.so: %.xo
-	$(LD) -o $@ $^ $(SHOBJ_LDFLAGS) $(LDFLAGS) $(LIBS)
+	$(CC) -o $@ $^ $(SHOBJ_LDFLAGS) $(LDFLAGS) $(LIBS)
 
 .PHONY: clean
 
